src/Symfony/Component/VarDumper/Caster/DOMCaster.php

```diff
@@ -138,16 +138,16 @@ public static function castDocument(\DOMDocument $dom, array $a, Stub $stub, boo
             'doctype' => $dom->doctype,
             'implementation' => $dom->implementation,
             'documentElement' => new CutStub($dom->documentElement),
-            'actualEncoding' => $dom->actualEncoding,
+            'actualEncoding' => $dom->encoding,
```

Comment:
what about removing the legacy properties instead? casters don't need to report useless data

Reply:
I was not sure about how this could be a breaking change thus this update, but I missed the `@internal` annotation on the class. Definitely better